src/Symfony/Component/Console/Output/Output.php

```diff
@@ -17,13 +17,14 @@
 /**
  * Base class for output classes.
  *
- * There are five levels of verbosity:
+ * There are six levels of verbosity:
  *
  *  * normal: no option passed (normal output)
  *  * verbose: -v (more output)
  *  * very verbose: -vv (highly extended output)
  *  * debug: -vvv (all debug output)
- *  * quiet: -q (no output)
+ *  * quiet: -q (only output errors)
+ *  * silent: --silent (no output)
```

Comment:
```suggestion
 *  * silent: (no output)
```
As the second column only shows shortcuts, we should not add `--silent` here, should we?

Reply:
This is a list of the available verbosities and how to activate them using CLI options. Not showing one because there is no single-letter variant is much worse than showing one longer option imho.